pkg/expr/func_test.go

```diff
@@ -52,20 +52,21 @@ func TestEQFunc(tt *testing.T) {
 		{"svc1.ns1.cluster", "*.ns1.cluster1", false},
 		{net.ParseIP("10.3.25.1"), net.ParseIP("10.3.25.1"), true},
```

Comment:
How was this test case working prior to the fix ?

Reply:
![Magic](https://media.giphy.com/media/12NUbkX6p4xOO4/giphy.gif)

Reply:
This test works before because all the IP address test goes through the "default": deepEqual. This explains why that chunk of code shows "not hit" in test coverage.